third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -25,7 +25,9 @@ std::vector<bool> computeContiguity(
   TORCH_CHECK(
       sizes.size() == strides.size(),
       "compute_contiguity: Sizes and strides must have the same number of dimensions");
-  auto not_broadcast = [&](auto i) { return strides[i] != 0 && sizes[i] != 1; };
+  auto not_broadcast = [&](auto i) {
+    return (sizes[i] != 1) || (strides[i] != 0 && sizes[i] != 1);
```

Comment:
Probably this should be `sizes[i] != 1`? `test_compute_contiguity` might fail if we do so, but we can skip that test for now.